main_hall.py

```diff
@@ -0,0 +1,92 @@
+class Main_Hall:
+    floor1_is_flooded = True
+    if floor1_1_flooded = True:
+        def __int__(self,global_state,inventory):
+            self.global_state = global_state
+            self.inventory = inventory
+            self.set_state()
+            self.description = """ You are the end end of senior hall near the library.
+                               To the north there are stairs leading down to the main hall which looks flooded,
+                               to the south there is a door that leads to the ASL room,
+                               to the west is the history hall,
+                               to the east is the senior hall. Choose wisely.. """
+            print(self.description())
+            self.run()
+            
+
+        def run():
+            self.route(input(">>"))
+    
+        def route(command):
+            if command in ["go north", "go south", "go east", "go west"]:
+                move(command.split(" ")[1])
+            else:
+                if command in ["Walk down stairs", "Enter stairs", "pass through stairs,"]:
+                    print("You cannot enter! The water is too deep to pass!")
+                else:
+                    print("What???")
+                self.run()
+
+        def move(direction):
+            for key in self.state.keys():
+                self.global_state[self.name+"_"+key] = self.state[key]
+            if direction == "north":
+                print("The stairs down to the art room seem to be flooded")
+            if direction == "west":
+                room = SeniorHall(self.global_state,self.inventory)
+                room.run()
+            if direction = "south":
+                print("The doors to the ASL room seem to be locked. You look into the window but fail to see anything for it is too dark")
```

Comment:
Yeah that's the only reason I did triple quotes. Just so I could see what ones doing while working within the window.

Sent from my iPhone

> On Jun 14, 2015, at 8:25 PM, Brayden <notifications@github.com> wrote:
> 
> In main_hall.py:
> 
> > +                if command in ["Walk down stairs", "Enter stairs", "pass through stairs,"]:
> > +                    print("You cannot enter! The water is too deep to pass!")
> > +                else:
> > +                    print("What???")
> > +                self.run()
> > +
> > +        def move(direction):
> > +            for key in self.state.keys():
> > +                self.global_state[self.name+"_"+key] = self.state[key]
> > +            if direction == "north":
> > +                print("The stairs down to the art room seem to be flooded")
> > +            if direction == "west":
> > +                room = SeniorHall(self.global_state,self.inventory)
> > +                room.run()
> > +            if direction = "south":
> > +                print("The doors to the ASL room seem to be locked. You look into the window but fail to see anything for it is too dark")
> Triple quotes does NL to space conversion?
> 
> —
> Reply to this email directly or view it on GitHub.
>